pkg/quarterdeck/users_test.go

```diff
@@ -20,7 +20,7 @@ func (s *quarterdeckTestSuite) TestUserDetail() {
 
 	// Test passing empty ULID results in StatusNotFound error
 	user, err := s.client.UserDetail(ctx, "")
-	s.CheckError(err, http.StatusNotFound, "resource not found")
+	s.CheckError(err, http.StatusUnauthorized, "this endpoint requires authentication")
```

Comment:
I noticed this test failed and realized that this is the correct error that should be returned since the requester does not have the correct permissions.  I am not sure how this had passed before.